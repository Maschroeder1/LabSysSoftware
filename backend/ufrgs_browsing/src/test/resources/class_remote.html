


<html>
<head><title>UFRGS - Horários e Turmas</title>
    <link href="/Design/formulario.css" rel="stylesheet" type="text/css" />
    <link href="/Design/simples.css" rel="stylesheet" type="text/css" />
    <link href="/Design/texto.css" rel="stylesheet" type="text/css" />
    <link href="/Design/tabela.css" rel="stylesheet" type="text/css" />
</head>

<body>


<!-- Aqui inicio o conteudo desta página -->


<fieldset class="moldura fieldset-1">
    <legend class="legend-2">&nbsp;&nbsp;Horários e Turmas - 2022/2&nbsp;&nbsp;</legend>

    <table class="modelo1" style="width:95%">
        <tr valign="middle" >
            <td class="th1" align="center" width="35%">Atividade de Ensino</td>
            <td class="th1" align="center">Carga<br>Horá-<br>ria</td>
            <td class="th1" align="center">Tur-<br>ma</td>
            <td class="th1" align="center">Vagas<br>Ofere-<br>cidas</td>
            <td class="th1" align="center" width="27%">Horário-Período-Local</td>
            <td class="th1" align="center">Professor(es)</td>
        </tr>

        <tr class="modelo1odd" valign="top"  style="font-size:12px ">
            <td align="left">&nbsp;ECONOMIA A&nbsp;</td>
            <td align="center">&nbsp;4&nbsp;</td>
            <td align="center">&nbsp;D &nbsp;</td>
            <td align="center">&nbsp;30&nbsp;</td>
            <td align="left"><li style="color:#006699; margin-bottom: 5px;">Ensino a Distância</li><li>Terça 16:30 - 18:10 (2)<br> </li><li>Quinta 16:30 - 18:10 (2)<br> </li><li><b></b></li></td>
            <td align="left"><li>Maurício Andrade Weiss</li></td>
        </tr>
    </table>
</fieldset>




<!-- Aqui encerra o conteudo desta página -->

</body>
</html>
