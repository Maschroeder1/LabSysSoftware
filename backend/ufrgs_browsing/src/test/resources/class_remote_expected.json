{
  "timeslots": [
    {
      "classIdentifier": "D",
      "availableSlots": 30,
      "professors": [
        "Maur�cio Andrade Weiss"
      ],
      "scheduledTimes": [
        {
          "day": "Ter�a",
          "shortDay": "ter",
          "startTime": "16:30",
          "endTime": "18:10",
          "location": "EAD",
          "locationMap": null
        },
        {
          "day": "Quinta",
          "shortDay": "qui",
          "startTime": "16:30",
          "endTime": "18:10",
          "location": "EAD",
          "locationMap": null
        }
      ]
    }
  ],
  "credits": 4
}